tests/docker-compose.test.yml

```diff
@@ -24,6 +24,6 @@ services:
     image: goharbor/redis-photon:4.0
     restart: always
     volumes:
-      - /data/redis:/data
+      - /data/redis:/var/lib/redis
```

Comment:
So redis in `docker-compose.test.yml` was not working previously?

Reply:
it's working previously, just not use a sharable folder
and actually this redis is using wrong tag, it should not be 4.0

Reply:
actually we should not use any data volume in such test yml, it will make the test env dirty and can't be reusable. We can fix this later